app/resources/forms.js.template

```diff
@@ -376,13 +376,15 @@ function make_form_expandable(expandedSection) {
 }
 
 // Show the side menu
+// Change the sidenav maxWidth for animation
```

Comment:
This comment should be just before the line to change maxWidth. The comment here should just contain general comment for the entire function.